frontend/components/Input/Input.js

```diff
@@ -0,0 +1,28 @@
+import React, { forwardRef } from "react";
+import styled from "@emotion/styled";
+import Text from "../Text/Text";
+import { color, fontWeight, spacing } from "../../theme";
+
+const StyledLabel = styled.label`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledInput = styled.input`
+  border-radius: ${spacing["0.5"]};
+  padding: ${spacing["0.5"]} ${spacing["0.75"]};
+  font-family: "IBM Plex Sans", sans-serif;
+`;
+
+const Input = forwardRef(({ id, labelText, errorMessage, ...props }, ref) => {
```

Comment:
Please elaborate on the approach used here

Reply:
ForwardRef allows other components to have access to the ref that controls the underlying child element. In this case, I wanted the ref from the useForm hook to have control over the input element. Using ForwardRef makes this a possibility.